plugins/inputs/logfile/tail/tail_test.go

```diff
@@ -8,8 +8,12 @@ import (
 	"strings"
 	"testing"
 	"time"
+
+	"gotest.tools/assert"
```

Comment:
This looks like a different assert package than what we normally import. Not sure if this was intentional. Didn't see an update to the go.mod or go.sum so I'm not too bothered but wanted to call it out.

Reply:
Vscode auto import.
I will fix it.
Good catch.